csharp/src/Ice/Communicator-PluginManager.cs

```diff
@@ -178,7 +163,7 @@ private void LoadPlugins(ref string[] cmdArgs)
                 }
             }
 
-            // Load and initialize the plug-ins defined in the property set with the prefix "Ice.Plugin.". These
+            // Loads and activates the plug-ins defined in the property set with the prefix "Ice.Plugin.". These
```

Comment:
Fix the comment, we no longer initialize/activate the plugins here

Reply:
Fixed.